scripts/github-actions/get-sub-directory-names.js

```diff
@@ -0,0 +1,13 @@
+/**
+ * @param {typeof import("fs")} fs
+ * @param {string} dir
+ */
+function getSubdirNames(dir) {
+  const fs = require("fs");
```

Comment:
Should `fs` remain a parameter and be passed from the caller?

Reply:
Updated here:
https://github.com/Azure/bicep-registry-modules/commit/20d412d9da3b54faf759004a8783918aee430fe4

Thanks!